pickfirst.go

```diff
@@ -81,38 +95,50 @@ func (b *pickfirstBalancer) UpdateClientConnState(cs balancer.ClientConnState) e
 			return balancer.ErrBadResolverState
 		}
 		b.state = connectivity.Idle
-		b.cc.UpdateState(balancer.State{ConnectivityState: connectivity.Idle, Picker: &picker{result: balancer.PickResult{SubConn: b.sc}}})
-		b.sc.Connect()
+		b.cc.UpdateState(balancer.State{
+			ConnectivityState: connectivity.Idle,
+			Picker:            &picker{result: balancer.PickResult{SubConn: b.subConn}},
+		})
+		b.subConn.Connect()
 	} else {
-		b.cc.UpdateAddresses(b.sc, cs.ResolverState.Addresses)
-		b.sc.Connect()
+		b.cc.UpdateAddresses(b.subConn, cs.ResolverState.Addresses)
+		b.subConn.Connect()
 	}
 	return nil
 }
 
-func (b *pickfirstBalancer) UpdateSubConnState(sc balancer.SubConn, s balancer.SubConnState) {
+func (b *pickfirstBalancer) UpdateSubConnState(subConn balancer.SubConn, s balancer.SubConnState) {
```

Comment:
I was OK with the parameter name being short, since the scope is more limited.  But if you're changing that, let's also change `s` to `scs` or `subConnState`?